reference/fleet/providers.md

```diff
@@ -1,25 +1,42 @@
 ---
-navigation_title: Providers
+navigation_title: Agent providers
 mapped_pages:
   - https://www.elastic.co/guide/en/fleet/current/providers.html
+applies_to:
+  stack: ga
 products:
   - id: fleet
   - id: elastic-agent
 ---
 
-# Configure providers for standalone {{agent}}s [providers]
+# {{agent}} providers [providers]
 
 
-Providers supply the key-value pairs that are used for variable substitution and conditionals. Each provider’s keys are automatically prefixed with the name of the provider in the context of the {{agent}}.
+Providers supply the key-value pairs that are used for variable substitution and conditionals. Each provider's keys are automatically prefixed with the name of the provider in the context of the {{agent}}.
 
-For example, a provider named `foo` provides `{"key1": "value1", "key2": "value2"}`, the key-value pairs are placed in `{"foo" : {"key1": "value1", "key2": "value2"}}`. To reference the keys, use `{{foo.key1}}` and `{{foo.key2}}`.
+For example, if a provider named `foo` provides `{"key1": "value1", "key2": "value2"}`, the key-value pairs are placed in `{"foo" : {"key1": "value1", "key2": "value2"}}`. To reference the keys, use the variables `${foo.key1}` and `${foo.key2}`.
 
 
-## Provider configuration [_provider_configuration]
+## Provider usage by deployment model [provider_usage_by_deployment_model]
 
-The provider configuration is specified under the top-level `providers` key in the `elastic-agent.yml` configuration. All registered providers are enabled by default. If a provider cannot connect, no mappings are produced.
+How you use providers depends on whether you're running a standalone or a {{fleet}}-managed {{agent}}.
 
-The following example shows two providers (`local` and `local_dynamic`) that supply custom keys:
+### Standalone {{agent}}
+
+On standalone {{agent}}, providers can be configured through the `providers` key in the `elastic-agent.yml` configuration file. You can enable, disable, and configure provider settings as needed. For more details, refer to [Provider configuration](#provider_configuration).
+
+### {{fleet}}-managed {{agent}}
```

Comment:
This is nice work, @vishaangelova. 
A couple of things while we wait for reviews from Dev...

Consider making "Providers" part of the headings--like "Configure providers for standalone agent and "Configure providers for Fleet-managed agent." We get better SEO results when key words are adjacent. If one of these topics comes up in search results, the user might be more likely to click if the heading pro to answer their question.  

Also some new headings need anchors defined.